dartagnan/src/main/java/com/dat3m/dartagnan/wmm/axiom/Axiom.java

```diff
@@ -47,13 +51,21 @@ public Relation getRelation() {
         return rel;
     }
 
+    public boolean isFlagged() {
+        return flag;
+    }
+
```

Comment:
I think the name `flagged` is non-descriptive. We should find a better name to indicate the meaning of this.

Reply:
I think we should use `flag` to be consistent with herd. However I do agree the name is not very descriptive so I added some documentation

Reply:
I don't think the internal name we give to the variable has to match the syntax. We can keep `flag` as syntax in the`.cat` file and still use some more descriptive name internally. But I'm also fine with the comment for now.